spec/acceptance/standard_spec.rb

```diff
@@ -6,7 +6,11 @@
     # Using puppet_apply as a helper
     it 'should work with no errors based on the example' do
       pp = <<-EOS
-        class { 'hashi_stack::repo': } -> package { 'packer': ensure => installed }
+        include hashi_stack::repo
+        package { 'packer':
+          ensure  => installed,
+          require => Class['Hashi_stack::Repo'],
+        }
```

Comment:
Btw, in the future I hope to expand voxpupuli-acceptance I hope to make it easier here to use example files. In theforeman we have [shared examples](https://github.com/theforeman/puppet-foreman/blob/c3c090ce8645a3f3da9cf7290ac13cab1503d8c6/spec/spec_helper_acceptance.rb#L35-L52) for that.